javav2/example_code/acmpca/src/main/java/com/example/acmpca/scenarios/CreatePrivateCertificateAuthorityAD.java

```diff
@@ -0,0 +1,244 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.acmpca.scenarios;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.regions.Region;
+import java.util.Arrays;
+import java.util.List;
+
+import software.amazon.awssdk.services.acmpca.AcmPcaClient;
+import software.amazon.awssdk.services.acmpca.model.AcmPcaException;
+import software.amazon.awssdk.services.acmpca.model.ASN1Subject;
+
+import software.amazon.awssdk.services.acmpca.model.CertificateAuthorityConfiguration;
+import software.amazon.awssdk.services.acmpca.model.CertificateAuthorityType;
+
+import software.amazon.awssdk.services.acmpca.model.CreateCertificateAuthorityRequest;
+import software.amazon.awssdk.services.acmpca.model.CreateCertificateAuthorityResponse;
+import software.amazon.awssdk.services.acmpca.model.CustomAttribute;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateAuthorityCsrRequest;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateAuthorityCsrResponse;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateRequest;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateResponse;
+import software.amazon.awssdk.services.acmpca.model.ImportCertificateAuthorityCertificateRequest;
+
+import software.amazon.awssdk.services.acmpca.model.IssueCertificateRequest;
+import software.amazon.awssdk.services.acmpca.model.IssueCertificateResponse;
+import software.amazon.awssdk.services.acmpca.model.KeyAlgorithm;
+import software.amazon.awssdk.services.acmpca.model.SigningAlgorithm;
+import software.amazon.awssdk.services.acmpca.model.Validity;
+import software.amazon.awssdk.services.acmpca.waiters.AcmPcaWaiter;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo;
+import org.bouncycastle.cert.jcajce.JcaX509ExtensionUtils;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.pkcs.PKCS10CertificationRequest;
+import org.bouncycastle.util.io.pem.PemReader;
+import java.util.Base64; 
+
+import lombok.SneakyThrows;
```

Comment:
1. Avoid using `SneakyThrows` for public code 
2. You don't actualy use this - maybe use an autoformatter tool to help you look through all the other files and see if there are other unused imports